### ISSUE TITLE
Can’t Access HomeBridge Devices Over WAN

### ISSUE THREAD

**Issue by jungopro** [nfarina/homebridge#1845]
Hello
All HB accessories work as expected over LAN. When I’m away from home they all show up as non-responding (including the HB device itself)
I have both AppleTV & iPad at home to enable remote access

Also, Philips hue bulbs that are connected to HomeKit via hue bridge (not HomeBridge) are working remotely as expected

Any idea?

**Comment by jungopro** [comment]
Hi

The issue is fixed for me, I'm posting it here if anyone else may need assistance:

A complete reset of the Apple TV resolved the issue, I can now access all Homebridge connected devices via LAN or WAN.

As a side note, I also did a reset to my iphone a few days ago to resolve an issue when after a few minutes homebridge would show as "No response" in Home App (via LAN) and this also resolved the issue
And today, I did a reset to my ipad to solve the same issue on the ipad...

Bottom line - **as sadly as it may be** - a complete reset and restore from backup is the only thing that help me resolve those "no response" issues
